server/service/osquery_utils/queries.go

```diff
@@ -1018,6 +1064,10 @@ func GetDetailQueries(ac *fleet.AppConfig, fleetConfig config.FleetConfig) map[s
 		generatedMap["users"] = usersQuery
 	}
 
+	if !fleetConfig.Vulnerabilities.DisableWinOSVulnerabilities {
+		generatedMap["windows_update_history"] = windowsUpdateHistory
+	}
```

Comment:
Nit (not for tackling now) We could eventually introduce a new `Enabled func(fleetConfig)` field in `DetailQuery`.